Jenkinsfile_CNP

```diff
@@ -6,7 +6,7 @@ import uk.gov.hmcts.contino.GithubAPI
 def type = "java"
 def product = "civil"
 def component = "ccd"
-def camundaBranch = "master"
+def camundaBranch = "bug/CIV-4519"
```

Comment:
Don't forget to revert this before merge to master.

Reply:
Just reverted it to master :)